App/Program.cs

```diff
@@ -1,11 +1,16 @@
 using Microsoft.AspNetCore.Components.Web;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using SevenDTDWebApp;
+using SevenDTDWebApp.Services;
 
 var builder = WebAssemblyHostBuilder.CreateDefault(args);
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
+// Configure HttpClient with base address
 builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
 
+// Register API service
+builder.Services.AddScoped<IServerApiService, ServerApiService>();
```

Comment:
Will this allow the site to talk to the azure endpoints through CORS protections?